hack/deploy_fast.sh

```diff
@@ -40,3 +37,5 @@ else
     echo "Cluster config is not proper... deploying cluster and dpu operator"
     deploy_cluster_and_dpu_operator
 fi
+
+bash hack/deploy_traffic_flow_tests.sh
```

Comment:
the script should be made executable and have a shebang. Thereby it chooses its own interpreter, and you don't need to choose `bash` for it.